auto_builder.sh

```diff
@@ -60,6 +60,7 @@ echo "     5 - Atftp"
 echo "     6 - Powerman"
 echo "     7 - Conman"
 echo "     8 - iPXE roms"
+echo "     11 - Documentation"
 echo " 9 - Exit."
```

Comment:
I don't understand the numbering. Proposal:
9 - documentation
Q - Quit

Reply:
Yeah, I need to update it each time, boring. Q is a nice idea, I don't know why I was focusing on number...

Thanks a lot ! Let me update that.